src/os/bluestore/NVMEDevice.cc

```diff
@@ -843,18 +838,16 @@ void io_complete(void *t, const struct spdk_nvme_cpl *completion)
     // read submitted by AIO
     if(!task->return_code) {
       if (ctx->priv) {
-	if (!--ctx->num_running) {
+        if (!--ctx->num_running) {
```

Comment:
please drop the formatting-only changes.

Reply:
@tchaikov  When can the test be finished?

Reply:
if you are in hurry please post the link to teuthology result where one is able to access from public web. or you can apply for the sepia lab access to run it by yourself. otherwise you need to wait until this PR is included by a test batch.

i did include your change in my test batch, but found that this PR can be improved, so dropped it.

does this answer your q?